.github/workflows/render-site.yaml

```diff
@@ -0,0 +1,49 @@
+name: Check rendered site for broken links
+
+on:
+  push:
+    branches:
+      - main
```

Comment:
Does your workflow run fine without these lines? We don't allow pushing directly against `main` so this should technically not do anything. If so, I'd delete.